ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Certificate/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+##########################
+# Windows Code Sign Cert #
+##########################
+- name: Download Windows.p12
+  win_get_url:
+    url: https://github.com/AdoptOpenJDK/openjdk-installer/raw/master/windows.p12
+    dest: 'C:\Users\jenkins\windows.p12'
```

Comment:
It would be good to document where this comes from originally.

Reply:
this is fetched from our installer repo where the .p12 file lives

Reply:
I expect @jdekonin means document where the file came from originally before it was put into the installer repo.

I agree, though that's best documented in the installer repo alongside the file itself rather than here where the file is used.

Reply:
umm well it's our custom code signing certificate. Some of those details are in the secrets repo :)